source/content/wordpress-cron.md

```diff
@@ -1,56 +1,108 @@
+contributors: [whitneymeredith]
 ---
 
+## Cron for WordPress Overview
+
+Cron is a standard utility in Unix and Linux systems that it is used to schedule commands for automatic execution at configured intervals. These scheduled commands or tasks are known as Cron Jobs. The name Cron comes from the Unix system for scheduling jobs, ranging from once a minute to once a year. Cron is generally used for running scheduled backups, monitoring disk space, deleting files that are no longer required, running system maintenance tasks, and much more. WP-Cron must be disabled to allow Cron to schedule and run tasks.
+
 ## WP-Cron Overview
 
-WP-Cron executes specific tasks for WordPress powered sites. The name Cron comes from the Unix system for scheduling jobs, ranging from once a minute to once a year. Whether it's routine maintenance or scheduled alerts, any command that can be executed on Unix without user intervention can be scheduled as a Cron task.
+WP-Cron is a WordPress feature that only executes tasks when the page is loaded. WP-Cron is first loaded by WordPress when a page is requested on the front or backend of a site, at which point WP-Cron displays the necessary page to the site visitor. 
+
+Plugins and themes can add tasks to be executed at regular intervals. For example, if you have a plugin that scans Twitter for your tweets and then incorporates them into comments, it's most likely done with a WP-Cron job. WP-Cron opens up a whole new world of things that a WordPress powered site can do.
+
+## Understanding the Differences Between Cron and WP-Cron
+
+There are a few major differences between Cron for WordPress and WP-Cron.
+
+### When Jobs are Executed
+
+Cron is a system process that continuously searches for configured jobs to execute. In contrast, WP-Cron only runs when [someone visits the site](https://stackoverflow.com/questions/12895706/when-does-wp-cron-php-run-in-wordpress) because it is a web-based system. When someone navigates to your WordPress site, WP-Cron checks for jobs that need to be executed.
+
+<Alert title="Note" type="info">
+Cron will not execute jobs on inactive environments, including sleeping development environments and test environments.
+</Alert>
+
+### Types of Jobs Executed
 
-WP-Cron is similar in nature to Cron, but differs in a couple of very important ways. This feature is designed solely to handle WordPress routine tasks:
+The WP-Cron feature is designed solely to handle WordPress routine jobs:
 
  - Check for new version of the WordPress core, themes, and plugins
  - Clean up spam
 
-Plugins and themes can add tasks to be executed at regular intervals. For example, if you have a plugin that scans Twitter for your tweets and then incorporates them into comments, it's most likely done with a WP-Cron job.  WP-Cron opens up a whole new world of things that a WordPress powered site can do.
+In contrast, any command that can be executed on Unix without user intervention can be scheduled as a Cron job.
```

Comment:
We don't expose the full power of cron to the customer. Pantheon cron just runs WordPress cron from a periodic task, rather than as part of a page load.